pkg/daemon/server/daemon_server.go

```diff
@@ -305,6 +305,24 @@ func (ds *daemonServer) exposeMetrics(ctx context.Context) {
 			} else {
 				watermarkCmpNow.WithLabelValues(ds.pipeline.Name).Set(float64(time.Now().UnixMilli() - maxWM))
 			}
+
```

Comment:
Can we leave a comment here giving a small description of which metric we are updating here along with its use